spotinst/commons/common_organization_user_group.go

```diff
@@ -0,0 +1,117 @@
+package commons
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/spotinst/spotinst-sdk-go/service/organization"
+	"log"
+)
+
+const (
+	OrgUserGroupResourceName ResourceName = "spotinst_organization_user_group"
+)
+
+var OrgUserGroupResource *OrgUserGroupTerraformResource
+
+type OrgUserGroupTerraformResource struct {
+	GenericResource
+}
+
+type OrgUserGroupWrapper struct {
+	OrgUserGroup *organization.UserGroup
+}
+
+func NewOrgUserGroupResource(fieldsMap map[FieldName]*GenericField) *OrgUserGroupTerraformResource {
+	return &OrgUserGroupTerraformResource{
+		GenericResource: GenericResource{
+			resourceName: OrgUserGroupResourceName,
+			fields:       NewGenericFields(fieldsMap),
+		},
+	}
+}
+
```

Comment:
let's the hierarchy same across all resources: OnCreate, OnRead, OnUpdate

Reply:
changed in same order